gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -64,9 +64,8 @@ def test_image_pipe(tmpdir):
     mosaic = MosaicImage(image, energy_band=energy_band, offset_band=offset_band, data_store=data_store,
                          obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
     mosaic.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
-    # TODO: this fails on some machines, for unknown reasons.
-    #assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
-    #assert_almost_equal(mosaic.maps['bkg'].data.sum(), 1987.2088283446637)
-    #assert_almost_equal(mosaic.maps['exposure'].data.sum(), 5.9292301086257267e17)
-    #assert_almost_equal(mosaic.maps['significance'].lookup(center), 33.699068269123039)
-    #assert_almost_equal(mosaic.maps['excess'].data.sum(), 346.79117165533626)
+    assert_allclose(mosaic.maps['counts'].data.sum(), 2334.0, atol=3)
```

Comment:
@leajouvin Code wise I think it's OK to keep it like that for now, because maybe there isn't an easy solution to the problem. I was just bringing up the question, how the event binning is handled in general.